commands/label_errors.py

```diff
@@ -85,15 +87,24 @@ def main(
                 model_repository=model_repository,
             )
 
-        if content_input == "radio" and input_type == "IMAGE" and ml_task == "CLASSIFICATION":
+        assets = get_assets(
+            kili, project_id, asset_status_in, max_assets=max_assets, randomize=randomize_assets
+        )
+        asset_id_to_remove = []
+        for asset in assets:
+            label = get_label(asset, label_merge_strategy)
+            if (label is None) or (
+                ml_task == "CLASSIFICATION" and job_name not in label["jsonResponse"]
```

Comment:
why is it specific to classification ? If there is a good reason, please add it in a comment

Reply:
for everything but classification, the job_name may not appear if there is no annotation (no Object or Name Entity to detect). On the other end, we might remove this altogether because we could imagine a case where the image does not belong to any category (with the classification job not required).
I am not sure which is best: this or nothing?

Reply:
Ok, so I remove the second verification. I also changed the warnings message since job_name is no longer checked:
warnings.warn(f"${asset_id} removed because no labels where available")

Reply:
👍